app/pages/Map.css

```diff
@@ -31,22 +31,42 @@
       width: 300px;
     }
 
-    & #legend {
-      background-color: #F2F2F2;
-      display: inline-block;
-      height: 544px;
-      float: right;
-      width: 120px;
-    }
+    & .dropdown {
```

Comment:
I moved the dropdown CSS to `components/Selector.css` so that both the Profile and Map could share the styles. Can you reduce the specific styles here to only include the *exceptions* to the shared styles?